src/wallet/rpcwallet.cpp

```diff
@@ -2608,15 +2621,13 @@ UniValue listunspent(const UniValue& params, bool fHelp)
     LOCK2(cs_main, pwalletMain->cs_wallet);
 
     UniValue results(UniValue::VARR);
-    vector<COutput> vecOutputs;
-    pwalletMain->AvailableCoins(
-            vecOutputs,
+    vector<COutput> vecOutputs = pwalletMain->AvailableCoins(
             asOfHeight,
             false,        // fOnlyConfirmed
             nullptr,      // coinControl
             true,         // fIncludeZeroValue
             true,         // fIncludeCoinBase
-            false,        // fOnlySpendable
+            ISMINE_LEGACY_SPENDABLE,
```

Comment:
`z_listunspent` should be updated to return records for transparent UTXOs in addition to unspent shielded outputs.